test/ocsp/helper/helper.go

```diff
@@ -197,6 +197,44 @@ func getOCSPURL(cert *x509.Certificate) (*url.URL, error) {
 	return ocspURL, nil
 }
 
+// checkSignature checks the response has a valid signature with respect to `issuer`.
+//
+// If there is a delegated signer in the response, it checks that the delegated
+// certificate has a signature from `issuer`.
+//
+// It checks that the OCSP response is within the validity window of whichever
+// certificate signed it, and that that certificate is currently valid.
+func checkSignature(resp *ocsp.Response, issuer *x509.Certificate) error {
+	var ocspSigner = issuer
+	if delegatedSigner := resp.Certificate; delegatedSigner != nil {
```

Comment:
x/crypto/ocsp already does this [signature checking](https://github.com/golang/crypto/blob/70a84ac30bf9/ocsp/ocsp.go#L545-L571) for us, I think all we need to do is check that the certificate is still valid.